ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/embeddedActivityBlock/embeddedActivityBlock.component.ts

```diff
@@ -38,9 +38,8 @@ export class EmbeddedActivityBlockComponent implements OnInit {
             this.logger.logError(this.LOG_SOURCE, 'An error during completing an activity', err);
             return EMPTY;
         }),
-        take(1),
-        finalize(() => this.componentBusy.emit(false))
-    ).subscribe();
+        take(1)
+    ).subscribe(() => this.componentBusy.emit(false));
```

Comment:
Note: 
Submit parent activity request should be after **_successful_** submit request for EMBEDDED child instance only